File: 백준/Silver/2563. 색종이/색종이.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.util.StringTokenizer;

public class Main {
    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    static StringTokenizer st;
    static StringBuilder sb = new StringBuilder();

    static final int MAX = 101;
    static int N, ans = 0;
    static int[][] board = new int[MAX][MAX];

    public static void main(String[] args) throws Exception {
        run();
    }

    static void run() throws Exception {
        input();
        solve();
        print();
    }

    static void cover(int x, int y) {
        for (int i = y; i < y + 10; i++) {
            for (int j = x; j < x + 10; j++) {
                board[i][j] = 1;
            }
        }
    }

    static void input() throws Exception {
        N = Integer.parseInt(br.readLine());
        for (int i = 0; i < N; i++) {
            st = new StringTokenizer(br.readLine());
            int x = Integer.parseInt(st.nextToken());
            int y = Integer.parseInt(st.nextToken());
            cover(x, y);
        }
        br.close();
    }

    static void solve() {
        for (int i = 1; i < MAX; i++) {
            for (int j = 1; j < MAX; j++) {
                if (board[i][j] == 1)
                    ans++;
            }
        }
    }

    static void print() throws Exception {
        sb.append(ans + "");
        System.out.println(sb);
    }
}

File: 백준/Silver/2563. 색종이/색종이.cc
#include <cstring>
#include <iostream>
#define fio ios_base::sync_with_stdio(0), cin.tie(0), cout.tie(0)

using namespace std;

const int MAX = 101;
int N, ans;
int board[MAX][MAX];

void init() {
    ans = 0;
    memset(board, 0, sizeof(board));
}

void cover(int x, int y) {
    for (int i = y; i < y + 10; i++) {
        for (int j = x; j < x + 10; j++) {
            board[i][j] = 1;
        }
    }
}

void input() {
    cin >> N;
    for (int i = 0; i < N; i++) {
        int x, y;
        cin >> x >> y;
        cover(x, y);
    }
}

void sol() {
    int ni, nj;

    for (int i = 1; i <= 100; i++) {
        for (int j = 1; j <= 100; j++) {
            if (board[i][j] == 1) {
                ans++;
            }
        }
    }

    cout << ans;
}

void run() {
    init();
    input();
    sol();
}

int main() {
    fio;
    run();
    return 0;
}